Makefile

```diff
@@ -330,13 +326,19 @@ FLEET_DESKTOP_VERSION ?= unknown
 desktop-windows:
 	GOOS=windows GOARCH=amd64 go build -ldflags "-H=windowsgui -X=main.version=$(FLEET_DESKTOP_VERSION)" -o fleet-desktop.exe ./orbit/cmd/desktop
 
+deps-desktop-linux:
+ifeq ($(shell uname -s),Linux)
+	# Dependency required for Linux Fleet Desktop.
+	command -v apt-get && sudo apt-get install gcc libgtk-3-dev libayatana-appindicator3-dev -y || true
+endif
+
 # Build desktop executable for Linux.
 #
 # Usage:
 # FLEET_DESKTOP_VERSION=0.0.1 make desktop-linux
 #
 # Output: desktop.tar.gz
-desktop-linux:
+desktop-linux: deps-desktop-linux
```

Comment:
Actually this is not needed, because it all happens within the container (see `apt-get` command on `Dockerfile-desktop-linux`).